timemachine/cpp/src/kernels/k_fixed_point.cuh

```diff
@@ -70,27 +71,27 @@ template <typename RealType> unsigned long long __device__ __forceinline__ FLOAT
 
 /* FLOAT_TO_FIXED_ENERGY converts floating point energies into fixed point. Values beyond LLONG_MAX/LLONG_MIN
 * (or non-finite) will be capped to LLONG_MAX. This is accumulated into __int128 which handles the positive/negative overflows,
-* allows to account for overflows triggered by the summation of energies.
+* allows to account for overflows triggered by the summation of energies. EnergyType is a typedef for __int128.
```

Comment:
nit: comment at risk of becoming stale?

Reply:
I initially referenced `EnergyType` in this comment, but seemed specific to the choice of `__int128` so I simply referenced that `EnergyType` is `__int128` so that the reader could connect it with what the code states. I am alright with just referencing `__int128`, just seemed confusing to read `(EnergyType)accumulated_energy > LLONG_MAX  - Correctly detects that energies are beyond valid range`